### ISSUE TITLE
Wrong loss formula in [SOLUTION] Laboratory 2: Computer Vision Part 2: Debiasing Facial Detection Systems

### ISSUE THREAD

**Issue by hoangqnguyen** [aamini/introtodeeplearning#180]
In: https://github.com/MITDeepLearning/introtodeeplearning/blob/master/lab2/solutions/PT_Part2_Debiasing_Solution.ipynb

According to the description, the calculation should be: 
    total_loss = torch.mean(classification_loss + face_indicator * vae_loss)

Correct me if I'm wrong, Thanks!

![Image](https://github.com/user-attachments/assets/cdb26ac9-32a0-49b4-8e99-ae1db22904ac)
